src/App.js

```diff
@@ -1,12 +1,49 @@
-import React from 'react';
-import './App.css';
+import { useState, useEffect } from "react";
+import axios from "axios";
+import "./App.css";
+import PokemonCard from "./Components/PokemonCard";
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [pokemons, setPokemons] = useState(null);
+    const [pagina, setPagina] = useState(0);
+
+
+    useEffect(() => {
+        async function getPokemons() {
+            try {
+                const response = await axios.get(
+                    `https://pokeapi.co/api/v2/pokemon?limit=20&offset=${pagina}`
+                );
+
+                setPokemons(response.data.results);
+
+            } catch (error) {
+                alert("Oops, the pokemons got away");
+            }
+        }
+
+        getPokemons();
+
+    }, [pagina]);
+
+    return (
+<>
+    <header>
+        <img src="https://seeklogo.com/images/P/Pokemon-logo-497D61B223-seeklogo.com.png" alt="pokemonLogo" className="logo"/>
+    </header>
+       <div className="pageButtons">
+            <button className="pageButton1" disabled={pagina ===0} onClick={()=> setPagina(pagina-20)}>Vorige</button>
+            <button className="pageButton2" disabled={pagina ===1100} onClick={()=>setPagina(pagina +20)}>Volgende</button>
```

Comment:
Er mag een spatie na de vergelijking, dus `pagina === 1100`. En wat gebeurd er als we, voor één of andere rare reden, bij 1101 komen ipv 1100 omdat de makers een extra pokemon toevoegen? Misschien beter om de vergelijking _groter dan of gelijk aan_ te maken! Er mag ook weer een spatie tussen `() => setPagina(pagina + 20)` ipv `()=>setPagina(pagina +20`